inc/Logging/QueryMonitor/QueryMonitor.php

```diff
@@ -0,0 +1,52 @@
+<?php declare(strict_types = 1);
+
+namespace RemoteDataBlocks\Logging\QueryMonitor;
+
+use QM_Collectors;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+class QueryMonitor {
+	public static function init(): void {
+		add_filter( 'qm/collectors', [ __CLASS__, 'add_collectors' ], 90, 1 );
+		add_filter( 'qm/outputter/html', [ __CLASS__, 'add_outputters' ], 90, 1 );
+	}
+
+	public static function add_collectors( array $collectors ): array {
+		$collector_classes = [
+			'RemoteDataBlocks\Logging\QueryMonitor\RdbHttpRequestCollector',
```

Comment:
Was this, and `RdbHttpRequestOutputHtml` supposed to be pushed up as well? I'm not able to find them in the plugin

Reply:
Oops, yeah, not intentionally. That class will be in my next PR. No harm since there's a `class_exists` check below, but I'll fix.